kolibri/plugins/coach/assets/src/views/coach-exercise-render-page/index.vue

```diff
@@ -48,54 +58,58 @@
       'immersive-full-screen': require('kolibri.coreVue.components.immersiveFullScreen'),
       'content-renderer': require('kolibri.coreVue.components.contentRenderer'),
       'page-status': require('./page-status'),
-      'answer-history': require('./answer-history'),
-      'question-attempt': require('./question-attempt'),
+      'attempt-log-list': require('./attempt-log-list'),
+      'interaction-list': require('./interaction-list'),
+    },
+    data() {
+      return {
+        // TODO use state instead
+        currentInteractionIndex: 0,
+        currentAttemptLogIndex: 0,
+      };
+    },
+    watch: {
```

Comment:
This does not seem to be a watch property as I understand it.

Watch objects should map from a key of a watched property to either a function to invoke when the property changes, or a text string that is the name of a method on the component.